src/pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -224,7 +224,8 @@ class RichProgressBar(ProgressBarBase):
             If required `rich` package is not installed on the device.
 
     Note:
-        PyCharm users will need to enable “emulate terminal” in output console option in
+        Jupyter notebook users need to install extra dependencies with ``pip install rich[jupyter]``.
```

Comment:
How important is this? Should we check it in the code?

Reply:
(Sorry for the delay!)

Without these extra packages, users won't see any rich progress bar nor any errors. So, for notebook users, it's a very important note.

I'm not sure if it's feasible to check this in the code because `pip install rich[jupyter]` installs quite a few packages and checking them all in the code may lead to false positives/negatives when some of those packages get updated on their side.

Reply:
We can just check `if RequirementCache("rich[jupyter]"): print_warning_about_importance()"`

Reply:
@akihironitta